dockers/broad/arrays_picard_private/Dockerfile

```diff
@@ -1,20 +1,27 @@
 FROM frolvlad/alpine-glibc
 
-ENV TERM=xterm-256color
+LABEL MAINTAINER="Broad Institute DSDE <dsde-engineering@broadinstitute.org>"
 
-WORKDIR /usr/gitc
+ENV TERM=xterm-256color \
+    NO_VAULT=true \
+    ARTIFACTORY_URL=https://broadinstitute.jfrog.io/artifactory/libs-release-local/org/broadinstitute/picard-private
 
-# Install dependencies.
-RUN apk --update add bash curl findutils jq wget openjdk8-jre python2 unzip
+WORKDIR /usr/gitc
 
-# download the gsutil install script
-RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/install_google_cloud_sdk.bash -O - | bash
+ARG PICARD_PRIVATE_VERSION=61af9bff4587783e5981a496f422ea36102482b5
 
-# Set up Vault
-ENV NO_VAULT=true
-RUN curl https://releases.hashicorp.com/vault/1.0.2/vault_1.0.2_linux_amd64.zip > temp.zip \
-    && unzip temp.zip \
-    && rm temp.zip \
-    && mv vault /usr/local/bin/
 
-COPY . .
+# Install dependencies
+RUN apk add --no-cache bash curl findutils jq wget openjdk8-jre python2 unzip && \
```

Comment:
This will probably conflict with Cheyenne's PR to remove python 2 (just noting, it's fine as it)